firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/core/CrashlyticsController.java

```diff
@@ -105,6 +105,8 @@
   static final String FIREBASE_APPLICATION_EXCEPTION = "_ae";
   static final String FIREBASE_ANALYTICS_ORIGIN_CRASHLYTICS = "clx";
 
+  static final int REPORT_UPLOAD_VARIANT_LEGACY = 1;
```

Comment:
should we document what "legacy" means / changes?

Reply:
Let's chat offline about this.